docs/reference/commandline/kill.md

```diff
@@ -27,8 +27,11 @@ Options:
 
 ## Description
 
-The main process inside the container will be sent `SIGKILL`, or any
-signal specified with option `--signal`.
+The `docker kill` subcommand kills one or more containers. The main process
+inside the container will be sent `SIGKILL` (default), the stop-signal that is
+defined in the image using the [`STOPSIGNAL`](../../builder/#stopsignal)
+Dockerfile instruction, or any signal specified with option `--signal` or
+`STOPSIGNAL` . You can kill a container using the container's ID or name.
```

Comment:
I don't think this is correct, the `STOPSIGNAL` should only be used for `stop`, not `kill`. I just tested quickly on 17.05 and that's the case. Did this change in 17.06?